package.json

```diff
@@ -16,5 +16,6 @@
   },
   "devDependencies": {
     "typescript": "5.6.2"
-  }
+  },
+  "packageManager": "pnpm@8.6.6"
```

Comment:
This is just for my benefit since I'm now using [corepack](https://nodejs.org/api/corepack.html) to manage multiple `pnpm` installs.